### ISSUE TITLE
Java template fails on Windows

### ISSUE THREAD

**Issue by Doug-AWS** [aws/aws-cdk#1043]
I've run into issues trying to get the Java template working on my Windows laptop. The cdk.json file consists of:

```json
{
  "app": "/bin/bash ./app.sh"
}
```

This doesn't work on vanilla Windows, so I hacked out a batch file (app.bat):

@echo off
set /p class_path=<.classpath.txt
set class_path=.;.\target\classes;%class_path%;
call %JAVA_HOME%\bin\java.exe -cp %class_path% com.myorg.HelloApp hello-cdk %*

And changed cdk.json to:

```json
{
  "app": "app.bat"
}
```

But now I'm getting:

```
Exception in thread "main" java.lang.NoClassDefFoundError: software/amazon/awscdk/services/sqs/QueueRef
        at com.myorg.HelloApp.main(HelloApp.java:11)
Caused by: java.lang.ClassNotFoundException: software.amazon.awscdk.services.sqs.QueueRef
        at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:349)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
        ... 1 more
Subprocess exited with error 1
```

BTW, the Java template works like a champ on my Mac laptop. I don't have access to a Linux box right now, so I have no idea whether it works there.

**Comment by pedrobento988** [comment]
I was having the same problem as you.

I made a solution based on yours and I was able to successfully run it with the following "app.bat" content:
for /f %%a in ('type .classpath.txt') do %JAVA_HOME%\bin\java.exe -cp .;.\target\classes;%%a com.myorg.HelloApp hello-cdk %*


Apart from that, when I ran your command "set /p class_path=<.classpath.txt", it was truncating the env. variable value's to 1023/1024 characters. So it is very likely why you were missing some classes on classpath.

I hope it helps.
